http/response.go

```diff
@@ -89,13 +89,22 @@ func WriteErrorResponse(
 type ErrorResponseOptions struct {
 	ResponseOptions `json:"-"`
 
-	Err     error  `json:"-"`
-	ErrCode string `json:"errorCode"`
-	ErrData any    `json:"errorData,omitempty"`
+	RequestID string `json:"requestId"`
```

Comment:
omit empty?
when the using code does not set the request id / does not use a middleware that does that